go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/go-redis/redis v6.10.2+incompatible
 	github.com/go-yaml/yaml v2.1.0+incompatible
 	github.com/mr-tron/base58 v1.1.2
-	github.com/nspcc-dev/dbft v0.0.0-20200130105505-02c208d154bf
+	github.com/nspcc-dev/dbft v0.0.0-20200203090053-815df16967cb
```

Comment:
Could you run `go get dbft@master` again? There is another commit hash due to rebasing.